PULL_REQUEST_BAZEL_TARGETS

```diff
@@ -57,3 +56,23 @@
 
 # XNet compatibility test for changes to certifications
 *canonical_state/* //rs/tests/message_routing/xnet:xnet_compatibility
+
+*nns_delegation* //rs/tests/networking:nns_delegation_test
+
+rs/backup/*.rs //rs/tests/consensus/backup:backup_manager_upgrade_test
+               //rs/tests/consensus/backup:backup_manager_downgrade_test
+
+rs/recovery/src/*.rs //rs/tests/consensus/subnet_recovery:sr_app_same_nodes_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_same_nodes_enable_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_same_nodes_with_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_failover_nodes_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_failover_nodes_enable_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_failover_nodes_with_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_no_upgrade_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_no_upgrade_local_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_no_upgrade_enable_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_no_upgrade_with_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_app_large_with_chain_keys_test
+                     //rs/tests/consensus/subnet_recovery:sr_nns_same_nodes_test
+                     //rs/tests/consensus/subnet_recovery:sr_nns_failover_nodes_test
```

Comment:
Since you're including all subnet recovery tests you could as well write:
```suggestion
rs/recovery/src/*.rs //rs/tests/consensus/subnet_recovery:...
```